Cargo.toml

```diff
@@ -26,13 +26,22 @@ windows = { version = "0.62.2", features = [
     "Win32_System_Threading",
     "Win32_System_WindowsProgramming",
 ] }
+winit = "0.30.12"
 uuid = { version = "1.17.0", features = ["v4"] }
 serde = { version = "1.0.219", features = ["derive"] }
 serde_json = "1.0.140"
 git-version = "0.3"
 sysinfo = "0.32.1"
 steamlocate = "2.0.1"
 
+# test-app dependencies
+anyhow = "1.0"
+bytemuck = { version = "1.24", features = ["derive"] }
+env_logger = "0.10"
+log = "0.4"
+pollster = "0.3"
+wgpu = "27.0.0"
```

Comment:
kinda whacky we're introducing the wgpu dependency here, but I guess we're using the re-export from `egui-wgpu` in the main app?